extensions/denshobato/docs/ime-denshobato-spec-en.md

```diff
@@ -0,0 +1,645 @@
+    if (!values.imeText.trim()) {
+      await showHUD("❌ Please enter text");
+      return;
+    }
+
+    setIsLoading(true);
+    try {
+      await sendToTerminal(values.imeText);
+      await saveToHistory(values.imeText);
+      await loadHistory();
+      await showHUD("🕊️ Message sent successfully!");
+      setText(""); // Clear after sending
+    } catch (error) {
+      console.error("Send failed:", error);
+      await showHUD("❌ Failed to send message");
+    } finally {
+      setIsLoading(false);
+    }
+  }
+
+  async function handleClear() {
+    setText("");
+    await showHUD("🗑️ Cleared");
+  }
+
+  return (
+    <Form
+      isLoading={isLoading}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <Action.SubmitForm
+              title="Send"
+              icon={Icon.Terminal}
+              onSubmit={handleSubmit}
+              shortcut={{ modifiers: ["cmd"], key: "enter" }}
+            />
+            <Action
+              title="Clear"
+              icon={Icon.Trash}
+              onAction={handleClear}
+              shortcut={{ modifiers: ["cmd"], key: "k" }}
+            />
+          </ActionPanel.Section>
+          
+          {history.length > 0 && (
+            <ActionPanel.Section title="History">
+              {history.map((item, index) => (
+                <Action
+                  key={index}
+                  title={`${item.slice(0, 50)}${item.length > 50 ? "..." : ""}`}
+                  icon={Icon.Clock}
+                  onAction={() => setText(item)}
+                />
+              ))}
+            </ActionPanel.Section>
+          )}
+        </ActionPanel>
+      }
+    >
+      <Form.TextArea
+        id="imeText"
+        title="IME Text"
+        placeholder={`Enter multi-line IME text...\n\nExample:\nThis is line 1.\nThis is line 2.\nHello, world!`}
+        value={text}
+        onChange={setText}
+        info="cmd+Enter to send, cmd+K to clear"
+      />
+      
+      <Form.Description
+        title="Character Count"
+        text={`${text.length} characters${text.includes('\n') ? ` (${text.split('\n').length} lines)` : ''}`}
+      />
+      
+      <Form.Description
+        title="Usage"
+        text="• Enter multi-line IME text\n• cmd+Enter to send to active terminal\n• Automatically pastes via clipboard"
+      />
+    </Form>
+  );
+}
+```
+
+### 2. Clipboard Operations (src/utils/clipboard.ts)
+
+```typescript
+import { Clipboard } from "@raycast/api";
+import { execAppleScript } from "./applescript";
+
+export async function sendToTerminal(text: string): Promise<void> {
+  // Backup current clipboard content
+  const originalClipboard = await Clipboard.read();
+  
+  try {
+    // Set text to clipboard
+    await Clipboard.copy(text);
+    
+    // Execute cmd+v via AppleScript
+    await execAppleScript(`
+      tell application "System Events"
+        keystroke "v" using command down
+      end tell
+    `);
```

Comment:
Can use just use the [`Clipboard.paste`](https://developers.raycast.com/api-reference/clipboard#clipboard.paste) for better compatibility (Windows support)?